test/sql/copy/csv/test_csv_wrong_file.test_slow

```diff
@@ -0,0 +1,18 @@
+# name: test/sql/copy/csv/test_csv_wrong_file.test_slow
+# description: Test CSV read over wrong file format
+# group: [csv]
+
+require tpch
+
+# load the DB from disk
+load __TEST_DIR__/test_csv_tpch.db
+
+statement ok
+CALL dbgen(sf=1)
+
+restart
+
+statement error
+from read_csv('lineitem.db')
```

Comment:
This should probably be `__TEST_DIR__/test_csv_tpch.db`

Reply:
Absolutely.